src/ape_ethereum/transactions.py

```diff
@@ -235,13 +235,15 @@ def decode_logs(
         abi: Optional[
             Union[List[Union[EventABI, "ContractEvent"]], Union[EventABI, "ContractEvent"]]
         ] = None,
-    ) -> List[ContractLog]:
+    ) -> ContractLogContainer:
         if abi is not None:
             if not isinstance(abi, (list, tuple)):
                 abi = [abi]
 
             event_abis: List[EventABI] = [a.abi if not isinstance(a, EventABI) else a for a in abi]
-            return list(self.provider.network.ecosystem.decode_logs(self.logs, *event_abis))
+            return ContractLogContainer(
+                list(self.provider.network.ecosystem.decode_logs(self.logs, *event_abis))
```

Comment:
Since `ContractLogContainer` subclasses `list`, you can just do this now

```suggestion
                self.provider.network.ecosystem.decode_logs(self.logs, *event_abis)
```

Reply:
somehow this wasnt working actually that is why because it assumes type is just list which has no filter method

Reply:
now working